lib/models/database/user/user_data.dart

```diff
@@ -26,6 +30,7 @@ class UserData {
         username: data[usernameField],
         displayName: data[displayNameField],
         joinTime: data[joinTimeField],
+        centauriPoints: data[centauriPointsField],
```

Comment:
This is only a suggestion and we might want to discuss this with the team.
But given that we are adding the centauriPoints feature on top of the existing database, this means that some accounts won't have the `centauriPoints` field in their firestore document leading to an error.
Thus, we might want to check if the value is `null` and if so consider it as 0. We would have something like that:

```suggestion
        centauriPoints: data[centauriPointsField] ?? 0,
```